graalpy/graalpy-starter/pom.xml

```diff
@@ -32,14 +33,19 @@
     <dependency>
         <groupId>org.graalvm.polyglot</groupId>
         <artifactId>polyglot</artifactId> 
```

Comment:
This dependency is covered by `python-embedding`, I tend to omit it

Reply:
I didn't know...I'd prefer if is covered by the `python` dependency, so it's consistent across all languages. I'll leave this here for now.